src/storage-config/types.ts

```diff
@@ -13,4 +13,14 @@ export type ApplyOptions = {
    * Allows you to override the default storage config with a custom one
    */
   storageConfig?: StorageConfig.AsObject
+
+  /**
+   * Allows to import active on-chain deals to the Cid deals information.
+   */
+  importDealIds?: number[]
+
+  /**
+   * Allows to configure if a Job should ensure the new storage configuration.
+   */
+  noExec?: boolean
```

Comment:
New options for importing deals and deferring job execution now supported in js client.